src/components/BookList.jsx

```diff
@@ -3,46 +3,146 @@ import React from 'react';
+  };
+  const editStyle = {
+    marginRight: '.8em',
+    paddingRight: '.8em',
+    borderRight: '2px solid #e9e8e8',
+    color: '#4386bf',
+    cursor: 'pointer',
+  };
+  const deleteStyle = {
+    marginRight: '.8em',
+    paddingRight: '.8em',
+    borderRight: '2px solid #e9e8e8',
+    color: '#4386bf',
+    cursor: 'pointer',
+  };
+  const commentStyle = {
+    color: '#4386bf',
+    cursor: 'pointer',
+  };
+  const div1Style = {
+    display: 'flex',
+    gap: '1em',
+  };
+  const div1pStyle = { textAlign: 'center', color: 'gray' };
+  const div1spanStyle = { fontSize: '2.2em', color: 'black' };
+
   const {
     books, isLoading, errorMsg,
   } = useSelector((store) => store.book);
   const dispatch = useDispatch();
 
   if (isLoading) {
     return (
-      <div style={{ marginTop: '50px' }}>Loading...</div>
+      <div className="lds-roller">
+        <div />
+        <div />
+        <div />
+        <div />
+        <div />
+        <div />
+        <div />
+        <div />
+      </div>
     );
   }
 
   if (errorMsg) {
     return (
-      <div style={{ marginTop: '50px' }}>Failed to fetch</div>
+      <div className="error">Failed to fetch</div>
     );
   }
 
   return (
-    <ul>
+    <ul style={ulStyle}>
       {Object.entries(books).map(([id, book]) => book.map((bookItem) => (
-        <li key={id}>
+        <li
+          key={id}
+          style={liStyle}
+        >
           <div>
-            <p>{bookItem.category}</p>
-            <h2>{bookItem.title}</h2>
-            <p>{bookItem.author}</p>
+            <p style={catStyle}>
+              {bookItem.category}
+            </p>
+            <h2 style={titleStyle}>
+              {bookItem.title}
+            </h2>
+
+            <p style={authorStyle}>
+              {bookItem.author}
+            </p>
             <div>
-              <button type="button">edit</button>
               <button
                 type="button"
+                style={editStyle}
+              >
+                edit
+              </button>
+              <button
+                type="button"
+                style={deleteStyle}
                 onClick={() => {
                   dispatch(deleteBook(id));
                 }}
               >
                 delete
               </button>
-              <button type="button">comment</button>
+              <button
+                type="button"
+                style={commentStyle}
+              >
+                comment
+              </button>
             </div>
           </div>
-          <div>
-            <div />
+          <div className="progress">
+            <div style={div1Style}>
+              <div className="rotate">
+                <span className="circle"> </span>
+              </div>
+              <p style={div1pStyle}>
+                <span style={div1spanStyle}>
+                  64%
+                </span>
+                <br />
+                Completed
+              </p>
+            </div>
```

Comment:
- [ ] The circular percentage section is too closer to the current chapter section. Kindle move the circular percentage section to the left so there is more spacing between the two sections just as it's in the original design. 👍🏽

Reply:
Done 👌